blueprints/node-webkit/files/nw-runner.js

```diff
@@ -0,0 +1,112 @@
+var exec = require('child_process').exec;
+var NW_VERSION = '0.12.0';
+var os = require('os');
+var fs = require('fs');
+
+runNw('./node_modules/nw/bin/nw');
```

Comment:
Agreed. I'll have to pass './' as the project root but since we know nw-runner is ran from here it shouldn't be a problem